google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineExecutor.java

```diff
@@ -45,8 +45,14 @@ public void onStart(Process process) {
   @Override
   public void cancel() {
     if (process != null) {
-      process.destroy();
-      task.onCancel();
+      // Only destroy the process and signal cancellation if the process hasn't exited
+      // TODO(eshaul) replace this with Process#isAlive when switching to Java 8
+      try {
+        process.exitValue();
```

Comment:
need a comment on this line saying that process.exitValue() throws IllegalThreadStateException when the process has not exited